configs/alibaba-inc_clarity_design.json

```diff
@@ -53,12 +57,12 @@
     "text": ".markdown-body p, .markdown-body li"
   },
   "js_render": true,
-  "js_wait": 1,
```

Comment:
This value is needed given that your website takes time to render. Once you your website doesn't need client side rendering, you can remove this attribute. [See reference](https://community.algolia.com/docsearch/config-file.html#js_wait-optional)

Reply:
Added :)